addons/wp/functions/fnc_taskCQB.sqf

```diff
@@ -137,7 +142,7 @@ private _fnc_act = {
 // functions end ---
 
 // init
-params [ "_group", "_pos", ["_radius", 50], ["_cycle", 21] ];
+params ["_group", "_pos", ["_radius", 50], ["_cycle", 21], "_area", "_useWaypoint"];
```

Comment:
Same about last comment about `_area`. Should `_useWaypoint` get a default value here? It is being used in some condition.